submitted_models/marble_hd2_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -257,19 +257,21 @@
     </node>
 
     <!-- Thermal Camera -->
+    <!-- Note camera_info topic generated by ign-gazebo incorrectly removes
+         sensor name in topic string -->
     <node
       pkg="ros_ign_bridge"
       type="parameter_bridge"
       name="ros_ign_bridge_thermal_camera"
-      args="$(arg sensor_prefix)/thermal_camera/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
-      <remap from="$(arg sensor_prefix)/thermal_camera/camera_info" to="thermal_camera/camera_info"/>
+      args="$(arg sensor_prefix)/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
+      <remap from="$(arg sensor_prefix)/camera_info" to="thermal_camera/camera_info"/>
```

Comment:
Is it safe to say that the thermal camera will be the only camera who's `camera_info` topic is `$(arg sensor_prefix)/camera_info`? I know that we need this workaround since we can't change the topic in `ign-sensors`, but I just want to make sure that there will never be any other kind of cameras that would collide with this topic.

Reply:
yeah luckily the, normal camera, rgbd camera, and depth camera do not have this problem so they do not conflict with this `camera_topic` topic string here